lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1301,22 +1309,36 @@ def jsonrpc_help(self, p=None):
         """
 
         if not p:
-            return self._render_response(sorted(self.callable_methods.keys()), OK_CODE)
-        elif 'callable_during_start' in p.keys():
-            return self._render_response(self.allowed_during_startup, OK_CODE)
-        elif 'function' in p.keys():
-            func_path = p['function']
-            function = self.callable_methods.get(func_path)
-            return self._render_response(function.__doc__, OK_CODE)
+            return self._render_response(", ".join(sorted(self.callable_methods.keys())), OK_CODE)
+        elif 'callable_during_startup' in p:
+            return self._render_response(", ".join(sorted(self.allowed_during_startup)), OK_CODE)
```

Comment:
seems like help should return text. i added `jsonrpc_commands` to return a list of possible commands